src/mesh/checkpoint_io.C

```diff
@@ -76,6 +78,53 @@ void chunking(processor_id_type size, processor_id_type rank, unsigned int nspli
                              (1 - (int)nchunks) * std::numeric_limits<int>::max());
     }
 }
+
+std::string extension(const std::string & s)
+{
+  auto pos = s.rfind(".");
+  if (pos == std::string::npos)
+    return "";
+  return s.substr(pos, s.size() - pos);
+}
+
+std::string splitDir(const std::string & input_name, processor_id_type n_procs)
+{
+  return input_name + "/" + std::to_string(n_procs);
+}
+
+
+std::string headerFile(const std::string & input_name, processor_id_type n_procs)
```

Comment:
Ideally these helper functions should also not be camel-cased for consistency, but I don't care that much since they are just local.